cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -1072,6 +1115,128 @@ def cancel(self, req):
+            FileUploadDaemon(task).start()
+
+        _prepare_upload()
+        rsp.directUploadPath = self.get_direct_upload_path(req[http.REQUEST_HEADER]['Host'])
+        return jsonobject.dumps(rsp)
+
+    @replyerror
+    def direct_upload_file(self, req):
+        try:
+            UploadHandler(req, self.upload_tasks).handle_upload()
+        except Exception as e:
+            logger.exception("File upload failed: %s", str(e))
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)